src/components/organisms/tabs/OverviewTab/index.jsx

```diff
@@ -70,118 +71,123 @@ class OverviewTab extends React.Component {
 
     return (
       <>
-        {platformData && (
-          <MDBRow className="text-center text-md-left mb-4">
-            {platformData.statistic.languages.map((language, i) => {
-              if (i < 6) {
-                return (
-                  <MDBCol md="4" key={i}>
-                    <span className="mb-2 text-muted">
-                      <MDBIcon
-                        icon="square"
-                        className="pr-2"
-                        style={{ color: language.color }}
-                      />
-                      {language.name} <small>{language.share}%</small>
-                    </span>
-                  </MDBCol>
-                );
-              }
-            })}
-          </MDBRow>
-        )}
-        {true && (
-          <>
-            <MDBRow className="m-0 p-0">
-              <MDBCol md="6 text-left">
-                <p className="lead">Pinned</p>
-              </MDBCol>
-              <MDBCol md="6 text-right">
-                <span className="clickable text-muted">Customize</span>
-              </MDBCol>
-            </MDBRow>
-            <MDBRow className="pinned">
-              {pinned.map((item, i) => {
-                return <Pinned {...item} key={i} />;
+        {/* <ErrorBoundary hidden="true"> */}
```

Comment:
Can this be removed?
```suggestion
```

Reply:
No. I left this there to show the hidden flag.